kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -327,309 +327,279 @@
-            KAFKACLIENT_SSL_PROTOCOL_CONFIG,
-            ConfigDef.Type.STRING,
-            "TLS",
-            ConfigDef.Importance.MEDIUM,
-            KAFAKSTORE_SSL_PROTOCOL_DOC
-        )
-        .define(
-            KAFKACLIENT_SSL_PROVIDER_CONFIG,
-            ConfigDef.Type.STRING,
-            "",
-            ConfigDef.Importance.MEDIUM,
-            KAFAKSTORE_SSL_PROVIDER_DOC
-        )
-        .define(
-            KAFKACLIENT_SSL_CIPHER_SUITES_CONFIG,
-            ConfigDef.Type.STRING,
-            "",
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SSL_CIPHER_SUITES_DOC
-        )
-        .define(
-            KAFKACLIENT_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG,
-            ConfigDef.Type.STRING,
-            "",
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_KERBEROS_SERVICE_NAME_CONFIG,
-            ConfigDef.Type.STRING,
-            "",
-            ConfigDef.Importance.MEDIUM,
-            KAFKACLIENT_SASL_KERBEROS_SERVICE_NAME_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_MECHANISM_CONFIG,
-            ConfigDef.Type.STRING,
-            "GSSAPI",
-            ConfigDef.Importance.MEDIUM,
-            KAFKACLIENT_SASL_MECHANISM_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_KERBEROS_KINIT_CMD_CONFIG,
-            ConfigDef.Type.STRING,
-            "/usr/bin/kinit",
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SASL_KERBEROS_KINIT_CMD_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_KERBEROS_MIN_TIME_BEFORE_RELOGIN_CONFIG,
-            ConfigDef.Type.LONG,
-            60000,
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SASL_KERBEROS_MIN_TIME_BEFORE_RELOGIN_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_JITTER_CONFIG,
-            ConfigDef.Type.DOUBLE,
-            0.05,
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_JITTER_DOC
-        )
-        .define(
-            KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR_CONFIG,
-            ConfigDef.Type.DOUBLE,
-            0.8,
-            ConfigDef.Importance.LOW,
-            KAFKACLIENT_SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR_DOC
-        )
-        .define(
-            KAFKA_REST_RESOURCE_EXTENSION_CONFIG,
-            Type.LIST,
-            "",
-            Importance.LOW,
-            KAFKA_REST_RESOURCE_EXTENSION_DOC
-        );
+    return baseConfigDef(
+        KAFKAREST_PORT_DEFAULT,
+        KAFKAREST_LISTENERS_DEFAULT,
+        String.join("," , Versions.PREFERRED_RESPONSE_TYPES),
+        Versions.KAFKA_MOST_SPECIFIC_DEFAULT,
+        METRICS_JMX_PREFIX_DEFAULT_OVERRIDE
+    ).define(ID_CONFIG, Type.STRING, ID_DEFAULT, Importance.HIGH, ID_CONFIG_DOC)
+    .define(HOST_NAME_CONFIG, Type.STRING, HOST_NAME_DEFAULT, Importance.MEDIUM, HOST_NAME_DOC)
```

Comment:
nit: consistent formatting: i.e get these spread across multiple lines as the rest are.